infrastructure/quick-lint-js-web/apache2/sites-available/001-www.quick-lint-js.com.conf

```
@@ -0,0 +1,18 @@
+<VirtualHost *:80>
+	ServerName www.quick-lint-js.com${DEV_TLD}
+	RewriteEngine on
+	RewriteRule ^ https://quick-lint-js.com${DEV_TLD}%{REQUEST_URI} [END,NE,R=permanent]
+</VirtualHost>
+
+<VirtualHost *:443>
+	ServerName www.quick-lint-js.com${DEV_TLD}
+	RewriteEngine on
+	RewriteRule ^ https://quick-lint-js.com${DEV_TLD}%{REQUEST_URI} [END,NE,R=permanent]
+
+	SSLCertificateFile /etc/letsencrypt/live/quick-lint-js.com/fullchain.pem
```

Isn't this cert file only for `quick-lint-js.com`, not for `www.quick-lint-js.com`?

Shouldn't we use a different cert for this domain:
```suggestion
 	SSLCertificateFile /etc/letsencrypt/live/www.quick-lint-js.com/fullchain.pem
```

---

Nevermind. Certbot can create a cert for both domains in one file.